app/models/ppms.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module PPMS
+end
```

Comment:
its a rails thing for active support, it lets me use
```
class PPMS::Provider
end
```
instead of
```
module PPMS
  class Provider
  end
end
```
without it you would have to add ppms directory to the load paths